app/src/main/res/layout/fragment_profile.xml

```diff
@@ -9,12 +9,12 @@
     <ImageView
         android:id="@+id/imageView"
         android:layout_width="match_parent"
-        android:layout_height="100dp"
+        android:layout_height="@dimen/fragmentprofileImageview_height"
         android:layout_gravity="center"
-        android:layout_marginStart="16dp"
-        android:layout_marginTop="16dp"
-        android:layout_marginEnd="16dp"
-        android:layout_marginBottom="5dp"
+        android:layout_marginStart="@dimen/fragmentprofileImageview_marginStart"
+        android:layout_marginTop="@dimen/fragmentprofileImageview_marginTop"
+        android:layout_marginEnd="@dimen/fragmentprofileImageview_marginEnd"
+        android:layout_marginBottom="@dimen/fragmentprofileImageview_marginBottom"
         android:contentDescription="@android:string/yes"
```

Comment:
If this margin is used into some other screens maybe we can name this like medium_margin_start instead of fragmentprofileImageview_marginStart. 
The idea behind the dimension file is to reuse the dimensions as much as we can, so we need to name it in a very general manner.